Updated for build scan plugin v2.0

timyates opened the pull request:
### Context

2.0 of the scan plugin is available for Gradle 5.0

wolfs commented on subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/BuildScanPluginSmokeTest.groovy:
```diff
@@ -52,7 +52,8 @@ class BuildScanPluginSmokeTest extends AbstractSmokeTest {
         "1.15.2",
-        "1.16"
+        "1.16",
+        "2.0"
```
I guess the `@Ignore` from this test should be removed. I also think that only `2.0` is supported and no other version.

timyates replied:
Fixed, I removed the `GRACEFULLY_UNSUPPORTED` list and test, as we now seem to print a warning for any version prior to 2.0

timyates commented:
Added the tests back in, and fixed up where we were referring to `licenseAgreementUrl` and `licenseAgreementAgree` as they've now gone in favor of the new properties

🤞

@wolfs @facewindu Can you check I've done good? 😉

facewindu commented:
good to be merged

timyates commented:
Merged and merged `release` into `master`

Thanks all 👍
